src/components/stateTracker/js/StateTracker.js

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+var fluid_2_0_0 = fluid_2_0_0 || {};
+
+(function ($, fluid) {
+    "use strict";
+
+    fluid.registerNamespace("fluid.stateTracker");
+
+    fluid.defaults("fluid.stateTracker", {
+        gradeNames: ["fluid.component"],
+        events: {
```

Comment:
The component would be more usable if this field were modelised - as well as widening the contract of the "changeEvaluator" function to simply return any value which is then applied to the model. A change in that value in the opinion of the data binding system would then notify any modelListeners bound to the fie.d